### PR TITLE
Pritambranch

### PR DESCRIPTION
**Pull request by pritamgolder06**
compare it

### CODE REVIEW

**Review by nayeem-mubasshirul — CHANGES_REQUESTED** [review]
Add a feature that prints gold.

File: write.txt [hunk 1]
```diff
@@ -9,9 +9,10 @@ this is from mehedi
 this is develop branchhhh conflict by siraz
 What a project !! Lame project .
 bhai ignore me for forming the git group. what kind of joke is that.
-Pritam is writing this line.
+Pritam is Pritam  this line.
 i m adding new line from develop
 kgjergk
 ergkerjhioerjhiorejh 
-rhjkrehjrehiojohjre eyhuj54yui4390ygbjmthio rgyjyi
+Pritam, Make a confilct rgyjyi
```
Comment (pritamgolder06) [inline write.txt:16]: comment added

File: write.txt [hunk 2]
```diff
@@ -9,9 +9,10 @@ this is from mehedi
 this is develop branchhhh conflict by siraz
 What a project !! Lame project .
 bhai ignore me for forming the git group. what kind of joke is that.
-Pritam is writing this line.
+Pritam is Pritam  this line.
 i m adding new line from develop
 kgjergk
```
Comment (nayeem-mubasshirul) [inline write.txt:14]: spelling mistake

**Review by nayeem-mubasshirul — CHANGES_REQUESTED** [review]
fix random chars.

**Review by nayeem-mubasshirul — APPROVED** [review]
Thank you.

**Pull request merged**